src/GenericSpectator/Assets/Scripts/PlatformDeviceBootstrapper.cs

```diff
@@ -0,0 +1,49 @@
+﻿using UnityEngine;
```

Comment:
nit: new files should have the microsoft open source copy right header:

// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.

Reply:
Thanks!

---
In reply to: [337248381](https://github.com/microsoft/MixedReality-SpectatorView/pull/250#discussion_r337248381) [](ancestors = 337248381)